core/src/main/resources/reference.conf

```diff
@@ -127,4 +127,8 @@ dataRepoEntityProvider {
   bigQueryMaximumBytesBilled = 100GB
 }
 
-sam.timeout = 3 minutes
\ No newline at end of file
+sam.timeout = 3 minutes
+
+prometheus {
+  endpointPort = 9098
+}
```

Comment:
should we add the fastpass configs here too? i noticed they are only in test reference rn

Reply:
The FastPass configs are done in firecloud-develop, unfortunately.